macros/compare_all_columns.sql

```diff
@@ -0,0 +1,49 @@
+{% macro compare_all_columns(model_name, primary_key, prod_schema, exclude_columns) -%}
+  {{ return(adapter.dispatch('compare_all_columns', 'audit_helper')( model_name, primary_key, prod_schema, exclude_columns )) }}
+{%- endmacro %}
+
+{% macro default__compare_all_columns( model_name, primary_key, prod_schema, exclude_columns ) -%}
+
+  {% set relation = api.Relation.create(schema=prod_schema, identifier=model_name) %}
```

Comment:
@joellabes I'm curious if this addresses the `{schema_name}.{model_name}` concern, or if I've created the same problem in a different place here with `api.Relation.create(schema=prod_schema, identifier=model_name)`.

Reply:
@joellabes honestly, I think I was having a mental block on why this wouldn't work just fine. I've made the update!